.dockstore.yml

```diff
@@ -7,6 +7,10 @@ workflows:
     testParameterFiles:
           -  /warp/pipelines/skylab/optimus/example_inputs/human_v3_example.json
 
+  - name: snM3C
```

Comment:
we already have snM3C in the dockstore yml (around line 114) do you mind just removing those lines? 114-116

Reply:
Thank you! Just removed.